Domains/4-Application/Connectors/CivilInfrastructureFramework/Released/CifQuantityTakeoffs.01.00.02.ecschema.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="CifQuantityTakeoffs" alias="cifqto" version="01.00.02" description="iModel Bridge schema for common Quantity Takeoff data from Civil Infrastructure Framework (CIF) applications." xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
+    <ECSchemaReference name="BisCore" version="01.00.14" alias="bis"/>
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA"/>
+    <ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+    <ECSchemaReference name="QuantityTakeoffsAspects" version="01.00.02" alias="qto"/>
+    <ECSchemaReference name="CifUnits" version="01.00.06" alias="cifu"/>
+    <ECCustomAttributes>
+        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+            <SupportedUse>NotForProduction</SupportedUse>
```

Comment:
do you intend to release this schema s NotForProduction?

Reply:
Yes. I've changed it to Production.